sql-plugin/src/test/spark324/scala/com/nvidia/spark/rapids/shims/spark324/SparkShimsSuite.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*** spark-rapids-shim-json-lines
+{"spark": "324"}
+spark-rapids-shim-json-lines ***/
+package com.nvidia.spark.rapids.shims.spark324;
```

Comment:
nit: we keep replicating extra semicolon, for this or a future PR could get rid of all of them.
```
$ git grep -c '^package .*;' '*.scala' | wc -l
14
```

```suggestion
package com.nvidia.spark.rapids.shims.spark324
```

Reply:
I can create a cleanup PR for the other files